### PR TITLE
Add tests for Global Retention policy in hours

### PR DESCRIPTION
**Pull request by DHaussermann**
Add tests for Global Retention policy in hours

### CODE REVIEW

**Comment by DHaussermann** [comment]
@saturninoabril I still have this marked as draft because there are tests in the pipeline failing. 
Can you please provide guidance on what's happening here?
image

I'm not even sure if this is in reference to the 2 files I added as part of my PR

File: data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+# (Required) Ensure all values are filled up
+name: "Set global policy for messages in hours"
+status: Approved
```
Comment (saturninoabril) [inline data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md:4]:
```suggestion
status: Active
```

File: data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+labels: []
+tested_by_contributor: ''
+
+# (Optional) Test type and tools
+cypress: N/A
+detox: N/A
+mmctl: N/A
+playwright: N/A
+rainforest: []
+manual_test_environments: []
```
Comment (saturninoabril) [inline data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForAttachmentInHours.md:25]:
```suggestion
manual_test_environments:
```

File: data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+# (Required) Ensure all values are filled up
+name: "Set global policy for messages in hours"
+status: Approved
```
Comment (saturninoabril) [inline data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md:4]:
```suggestion
status: Active
```

File: data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+labels: []
+tested_by_contributor: ''
+
+# (Optional) Test type and tools
+cypress: N/A
+detox: N/A
+mmctl: N/A
+playwright: N/A
+rainforest: []
+manual_test_environments: []
```
Comment (saturninoabril) [inline data/test-cases/suite/system-console/compliance/data-retention-policy/SetGlobalPolicyForMessagesInHours.md:25]:
```suggestion
manual_test_environments:
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
There's an error validating the changes. I suggested the changes and should passed on validation.

Locally, you may run `deno task validate` and see it returns `All changes are valid.`

Screenshot 2024-09-13 at 10 17 55 AM

**Comment by DHaussermann** [comment]
@saturninoabril yes I see. I missed the `deno task validate` step and went straight to `deno task check` my mistake. 

But, I'm not seeing how tis get's me any closer, the error on my local looks the exact same :)

image

Where is the problem? 0/5 looks like Deno (maybe through a API) is making calls to remote files and something unexpected is in the response. Is this correct or am I misunderstanding the issue? 

Yesterday using the Brew package manager I updated Deno. I'm not clear on what I should do next.

**Comment by saturninoabril** [comment]
I think you're on track. Have you tried applying the change requests I commented? Once applied, you may try local validation again and see if it fixes the error you're seeing.

**Comment by DHaussermann** [comment]
@saturninoabril Awesome thank man!
image

Locally it's fine now so let's see. The pipeline should say the same.

**Comment by DHaussermann** [comment]
Implemented changes so default status is now correct and removed empty `[]` causing issue as the array I was trying to provide was not empty.

This is now ready for review.

**Comment by DHaussermann** [comment]
@saturninoabril can you please review once more when you have a chance?

**Review by saturninoabril — APPROVED** [review]
Thanks @DHaussermann, LGTM!

**Comment by saturninoabril** [comment]
@DHaussermann Test cases added as MM-T5627 and MM-T5628.

**Pull request merged**